server/images.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../database/index.js');
+
+module.exports.get = (req, res) => {
+  const id = req.params.song;
```

Comment:
What is song?

Reply:
Oops! lingering information/implementation from my old project!